src/send/hooks.ts

```diff
@@ -0,0 +1,41 @@
+import { TokenBalance, TokenBalanceWithAddress } from 'src/tokens/slice'
+import { PreparedTransactionsResult, prepareTransactions } from 'src/viem/prepareTransactions'
+import { TransactionRequestCIP42 } from 'node_modules/viem/_types/chains/celo/types'
+import { Address, encodeFunctionData } from 'viem'
+import erc20 from 'src/abis/IERC20'
+import BigNumber from 'bignumber.js'
+
+/**
+ * Prepare a transaction for sending an ERC-20 token.
+ *
+ * Works for ERC-20 and native transfers.
+ *
+ * NOTE: does not include fees for registering a DEK before sending a c-stable, since that is a different transaction.
+ *
+ */
+export async function prepareSendERC20Transaction(
```

Comment:
nit: should this be outside of `hooks` if it isn't a hook?

Reply:
I moved it to prepareTransactions, but then it felt weird having prepareSwapTransactions in a hooks module as well, so I moved that to prepareTransactions too